android/version.gradle

```diff
@@ -70,6 +70,7 @@ ext {
     VERSION_NAME=getVersionName()
     VERSION_ENVIRONMENT=getVersionEnvironment()
     VERSION_TAG=System.getenv("VERSION_TAG")
+    VERSION_GIT_TAG=System.getenv("VERSION_GIT_TAG")
```

Comment:
I think the IDE gradle sync keeps nulling this.

Following the getVersionEnvironment pattern, I had to add
```gradle
def getVersionGitTag = { ->
    String env_git_tag = System.getenv("VERSION_GIT_TAG")
    if (env_git_tag != null) {
        println "Using $env_git_tag from VERSION_GIT_TAG"
        return "$env_git_tag"
    } else {
        // Building probably from IDE
        println "Using release@$VERSION_NAME"
        return "release@$VERSION_NAME"
    }
}
```

and then set
```
VERSION_GIT_TAG=getVersionGitTag()
```

Reply:
Gotcha